lang/cs/Org.Apache.REEF.Network.Examples/GroupCommunication/PipelineBroadcastReduceDriverAndTasks/PipelinedBroadcastReduceDriver.cs

```diff
@@ -19,20 +19,16 @@
 using System.Collections.Generic;
```

Comment:
Why do we need to move Comm. group put of ...For IIMRU Driver we needs to introduce FT and hence those changes are necessary. Here we can leave them as they are. The problem I foresee is that ContextManager would need to be modified and made specific to IMRU in future and hence I feel we should let this piece of code as it is in term of organization.